bot/extensions/translator_cog.py

```diff
@@ -0,0 +1,38 @@
+from discord.ext.commands import Cog, hybrid_command
+from googletrans import Translator
+import googletrans
+from discord import Embed
+
+
+class TranslatorCog(Cog, name="Translator", description="Translate sentances from any languages to English."):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @hybrid_command(name='translator', help='Translate a sentence/word from any languages to English', \
```

Comment:
This is how it should be formatted (see bellow). The `\` is  unnecessary and the usage should go right under the name. 

```python
    @hybrid_command(name='translator', help='Translate a sentence/word from any languages to English',
                    usage="sentence={sentence}")
```

Reply:
Oh, yes. My IDE was messing with me when I did like you did for some reason lol. Thank you